src/lua/widget/label.c

```diff
@@ -30,26 +30,26 @@ static dt_lua_widget_type_t label_type = {
 static int label_member(lua_State *L)
 {
   lua_label label;
-  luaA_to(L,lua_label,&label,1);
+  luaA_to(L, lua_label, &label, 1);
   if(lua_gettop(L) > 2) {
-    const char * text = luaL_checkstring(L,3);
-    gtk_label_set_text(GTK_LABEL(label->widget),text);
+    const char * text = luaL_checkstring(L, 3);
+    gtk_label_set_text(GTK_LABEL(label->widget), text);
```

Comment:
@wpferguson please consider making lua module labels ellipsized by default. Currently some of the (long) labels in script manager dictate the minimum width of the left panel. I.e. the user can't set a narrower panel or if they did, it will jump around when the script manager is loaded or when it changes tabs. Adding
```
gtk_label_set_ellipsize(GTK_LABEL(label->widget), TRUE);
```
here mostly resolves this issue, but of course it's up to you to see if this is appropriate.

Long button labels have the same impact.

Reply:
See ff2890aca8c939889607f3641a827f0918a03a83